arch/arm/mach-imx/imx8m/Kconfig

```diff
@@ -27,6 +27,14 @@ config IMX8MP
 config SYS_SOC
 	default "imx8m"
 
+config SECONDARY_BOOT_RUNTIME_DETECTION
+	bool "SD/MMC sector offset runtime detection"
+	default n
+	depends on (IMX8MQ || IMX8MM) && SYS_MMCSD_RAW_MODE_U_BOOT_USE_SECTOR
```

Comment:
Should we depend on SPL_MMC_SUPPORT as well here?

Reply:
Wonder if needed because of SYS_MMCSD_RAW_MODE_U_BOOT_USE_SECTOR, but looking at Kconfig this is not checking for MMC either, so not sure.